vaccine/tests/test_utils.py

```diff
@@ -75,7 +75,6 @@ def test_search_hong_kong(self):
         assert Countries().search_for_country("hong kong")[0][0] == "HK"
 
     def test_search_ivory_coast(self):
-        assert Countries().search_for_country("ivory coast")[2][0] == "CI"
```

Comment:
This is the only change I'm concerned about. "Ivory Coast" is no longer a close match to "Republic of Côte d'Ivoire", which kind of makes sense, they are very different. Do we think people who live in Côte d'Ivoire will be typing "Ivory Coast"?

Reply:
No, I'm happy with this change. They can always adjust their search term.